playbooks/wordpress.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Deploy the OpenCraft.com server with wordpress
```

Comment:
@Agrendalath, this has to be generic and not specific to one particular site.

Reply:
Done.